src/components/HomepageStyles.scss

```diff
@@ -1,14 +1,135 @@
+.row {
+  padding-left:15px;
+  padding-right:0;
+}
+
+ul {
```

Comment:
I would avoid doing this and make it a classname or something instead. I recent found out that doing this will apply this style to ever `ul` on the site **facepalm**